src/parse_util.rs

```diff
@@ -1120,6 +1120,17 @@ pub fn parse_util_detect_errors_in_ast(
             if jcc.conjunction.has_source() && jcc.job.try_source_range().is_none() {
                 has_unclosed_conjunction = true;
             }
+            let continuations = jcc
+                .parent()
+                .and_then(|job_conj_list| job_conj_list.as_job_conjunction_continuation_list());
```

Comment:
should squash the two commits because the first one was wrong it seems

Reply:
Sure